src/components/containers.js

```diff
@@ -1,31 +1,46 @@
-/**
- * Components for laying out content within a consistently-styled container
- */
-
 import classnames from 'classnames';
 
 import { downcastRef } from '../util/typing';
 
+/**
+ * Components for laying out content within a consistently-styled container
+ */
+
 /**
  * @typedef {import('preact').ComponentChildren} Children
+ * @typedef {import('preact').JSX.HTMLAttributes<HTMLDivElement>} HTMLDivAttributes
+ * @typedef {import('preact').Ref<HTMLElement>} ElementRef
+ */
+
+/**
+ * @typedef CommonPresentationalProps
```

Comment:
"Presentational" feels a bit verbose but I don't have an immediate suggestion for an alternative.